src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -324,6 +334,35 @@ ov::pass::StateManagementPattern::StateManagementPattern(ParameterVector& kv_par
             if (alibi_slopes->get_element_type() == element::f32) {
                 alibi_slopes = std::make_shared<v0::Convert>(alibi_slopes, element::f32);
             }
+
+            // Jais-13b case
+            if (pattern_map.find(mirroring_abs) != pattern_map.end()) {
```

Comment:
Could you add unit tests for both cases: bloom alibi, jias alibi?

Reply:
We agreed to add these tests in the next PR.
We are going to take them from the real models, for that we need to improve PrintModel transformation first